node/web/src/lib.rs

```diff
@@ -119,18 +127,34 @@ async fn setup_node(
         .work_verifier_index(work_verifier_index.clone());
 
     // TODO(binier): refactor
-    if let Some(seed_nodes_url) = seed_nodes_url {
-        let peers = ::node::core::http::get_bytes(&seed_nodes_url)
-            .await
-            .expect("failed to fetch seed nodes");
-        node_builder.initial_peers(
-            String::from_utf8_lossy(&peers)
-                .split("\n")
-                .filter(|s| !s.trim().is_empty())
-                .map(|s| s.trim().parse().expect("failed to parse seed node addr")),
-        );
+    let mut all_raw_peers = seed_nodes_fixed.unwrap_or_default();
+    if let Some(seed_nodes_urls) = seed_nodes_urls {
```

Comment:
nit: I like when an empty line preceeds and follows anything with curly braces, thoughts? 

```suggestion

    if let Some(seed_nodes_urls) = seed_nodes_urls {
```

I wish this was something you could configure with rustfmt, but they wouldn't even review my PR... they just said it wasn't high enough priority 😞

Reply:
Up to you if you want to make this a standard -- I kinda like clustering related code together (e.g., in this case, we make a place to collect peers, then collect them into it, and finally push them into the node builder. Adding that space sorta breaks the semantic coupling imo)

Reply:
I just find the code harder to read when it's all clumped together.

Reply:
space added